### ISSUE TITLE
Character customization

### ISSUE THREAD

**Issue by Violet-Vibes** [yamashi/CyberEngineTweaks#57]
Might be out of scope of this mod/project, but being able to customize your character even after you started would be a nice feature.

**Comment by mikhnal** [comment]
One option no start: start_with_a_randomly_selected_character: true

**Comment by Brokeos** [comment]
> 
> 
> One option on start: start_with_a_randomly_selected_character: true

With this option we can customize or recreate our character ?

**Comment by mikhnal** [comment]
For me: customize, just for fun, like button to randomly create character when selecting a new game.

**Comment by lrhinehart0724** [comment]
I noticed the NCPD npcs have items we can get in the game like the vest, neck ,helment and glases utilty belt but the npc can wear them all at once. Like they have more equip slots then the player does idk. I tried the police uniform special outfit (https://www.youtube.com/watch?v=ZpCgaKFjyok) using this code "player.inventory.addItem(Items.SQ029_Police_Suit,)" but it has no utility belt and when you add the ncpd vest it doesn't have the neck brace. I'd just like to figure out a way to get a NCPD Full uniform.  Also being able to modify my owned car to a cop car would be sick. My top goals for this game at the moment is to achieve this gameplay.
